vendor/github.com/containernetworking/plugins/plugins/ipam/host-local/backend/disk/backend.go

```diff
@@ -56,7 +56,7 @@ func New(network, dataDir string) (*Store, error) {
 
 func (s *Store) Reserve(id string, ip net.IP, rangeID string) (bool, error) {
 	fname := filepath.Join(s.dataDir, ip.String())
-	f, err := os.OpenFile(fname, os.O_RDWR|os.O_EXCL|os.O_CREATE, 0600)
+	f, err := os.OpenFile(fname, os.O_RDWR|os.O_EXCL|os.O_CREATE, 0644)
```

Comment:
note that you're accidentally reverting the local patch to this file (which will then cause `verify-deps` to fail)

Reply:
hmm wonder how that happened, should i just ignore this change?

Reply:
Just make sure to undo this change (`git checkout vendor/github.com/containernetworking/plugins/plugins/ipam/host-local/backend/disk/backend.go`) after running `go mod vendor`. (Or I think if you use `make update-deps` to update the vendoring it will re-apply the patch for you?)

(What's going on here is that we needed to fix a CVE, but actually updating to the version of `containernetworking/plugins` that had the fix would have pulled in dozens of other dep updates, etc. So we just patched the fix into the vendor tree by hand, resulting in a vendor tree that no longer actually matches what `go.sum` thinks it should, but `make verify-deps` has a hack to deal with this (`./deps.diff`).)